apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/CommonCodings.java

```diff
@@ -0,0 +1,27 @@
+package gov.cms.bfd.server.war.commons;
+
+public enum CommonCodings {
+  MC(TransformerConstants.CODING_SYSTEM_HL7_IDENTIFIER_TYPE, "MC", "Patient's Medicare Number");
+
+  private final String system;
+  private final String code;
+  private final String display;
+
+  CommonCodings(String system, String code, String display) {
+    this.system = system;
+    this.code = code;
+    this.display = display;
+  }
+
+  public String getSystem() {
```

Comment:
Did you not use Lombok for all of the boilerplate because we don't have it available in this module?